core/src/main/java/com/sequenceiq/cloudbreak/controller/v4/StackV4Controller.java

```diff
@@ -478,8 +477,6 @@ public FlowIdentifier renewCertificate(Long workspaceId, String name, @Initiator
     public FlowIdentifier renewInternalCertificate(Long workspaceId, @TenantAwareParam String crn) {
         try {
             return stackOperationService.renewInternalCertificate(crn);
-        } catch (FlowNotTriggerableException e) {
-            throw new BadRequestException(e.getMessage());
```

Comment:
It is unrelated. These two lines add nothing to the code as all exceptions are caught right below this and handled the same way.
I simply modified this since I noticed the file giving me a warning about this.
I can revert but don't really see any reason to.